index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+
+<html>
+
+
+<head>
+
+<title>
+Lawrence's CTD portfolio
+</title>
+
+  <meta charset="UTF-8">
+  <meta name="description" content="CTD portfolio">
+  <meta name="keywords" content="HTML, CSS, JavaScript">
+  <meta name="author" content="Lawrence">
+</head>
+
+<body>
+Lawrence
+
+<section>
+<h1>About</h1>
+<p>I am learning Javascript so that I can learn to make React apps.</p>
+
+<section>
+<h1>Experience</h1>
+<p>I am currently attending Code the Dream bootcamp.</p>
+
+<section>
+<h1>Connect</h1>
+<p>
```

Comment:
This can vary based on the circumstances, but you are not required to wrap lists in paragraph (`<p></p>`) elements.